src/components/CommentComposer/CommentComposer.js

```diff
@@ -118,14 +169,32 @@ class CommentComposer extends PureComponent {
             rows='1'
             onChange={this.onChange}
           />
+          {maxLength && <div {...styles.maxLength}>
```

Comment:
Maybe a `renderProgress` method with an early `if (!maxLength) return null` could help reducing the repeated `maxLength` check. Adding a `remaining = maxLength - count` variable could make all other logic more obvious.